var/spack/repos/builtin/packages/r-sparsematrixstats/package.py

```diff
@@ -30,4 +38,7 @@ class RSparsematrixstats(RPackage):
     depends_on("r-rcpp", type=("build", "run"))
     depends_on("r-matrix", type=("build", "run"))
     depends_on("r-matrixstats", type=("build", "run"))
-    depends_on("r-matrixstats@0.60.0:", type=("build", "run"), when="@1.6.0:")
+    depends_on("r-matrixstats@0.60.0:0.63.0", type=("build", "run"), when="@1.6.0:1.12")
```

Comment:
I never remember if the upper bound is included or not, I wrote this under the assumption it's included.

Reply:
It is inclusive (https://spack.readthedocs.io/en/latest/packaging_guide.html#specifying-backward-and-forward-compatibility).